Can't start on Windows (Failed loading plugin file)

On Windows get these type of errors for default plugins:
`ERR! Plugins Failed loading plugin file <node_modules\E:\ssss\zone-mta-template\node_modules\zone-mta\plugins\core\default-headers>: Cannot find module 'E:\ssss\zone-mta-template\node_modules\E:\ssss\zone-mta-template\node_modules\zone-mta\plugins\core\default-headers'`

Looking at plugin_handle.js found next code (line 214 for version 1.13.3):
```javascript
                if (plugin.path.indexOf('/') < 0) {  
                    plugin.path = path.join(process.cwd(), 'node_modules', plugin.path);  
                }
```

In Windows you have to consider not "/", but "C:", "E:" etc (some letter + ":"). I think you can just check if second letter is ":".

Working code is:
```javascript
                if (plugin.path.indexOf('/') < 0 && plugin.path.indexOf(':') != 1) {
                    plugin.path = path.join(process.cwd(), 'node_modules', plugin.path);
                }
```

--- a/lib/plugin-handler.js
+++ b/lib/plugin-handler.js
@@ -211,7 +211,7 @@ class PluginHandler {
 
             try {
                 let loadStartTime = Date.now();
-                if (plugin.path.indexOf('/') < 0) {
+                if (plugin.path.indexOf('/') < 0 && plugin.path.indexOf('\\') < 0) {
                     plugin.path = path.join(process.cwd(), 'node_modules', plugin.path);
                 }
                 plugin.module = require(plugin.path); // eslint-disable-line global-require

